### ISSUE TITLE
[🐛] Plugin does not use composability API

### ISSUE THREAD

**Issue by Crevil** [snyk-tech-services/backstage-plugin-snyk#15]
- `node -v`: v14.15.4
- `npm -v`: 6.14.10
- OS: macOS
- Command run: N/A

### Expected behaviour

Adding the snyk plugin to a backstage installation with the [composability API](https://backstage.io/docs/plugins/composability).

When registering I would expect something along the lines of this.

```
<EntityLayout.Route path="/snyk" title="Snyk">
  <EntitySnykContent />
</EntityLayout.Route>
```

### Actual behaviour

Integration does not work as the plugin does not use the composability API and thus an `entity` is not available to inject into the widgets.

### Steps to reproduce

Follow the instructions in the README of this repo which uses the old API.
Most specifically when registering widgets.

```
// packages/app/src/components/catalog/EntityPage.tsx
const OverviewContent = ({ entity }: { entity: Entity }) => (
  <Grid container spacing={3} alignItems="stretch">
    ...
    <Grid item>        
      <SnykOverviewComponent entity={entity} />
    </Grid>
    ...
  </Grid>
);
```

**Comment by aarlaud** [comment]
I guess the proper plugin integration mechanism has been released now. thanks, we'll put it into our TODO.

**Comment by Crevil** [comment]
Yeah, we hit this as backstage it self has now fully been migrated which makes it impossible to use the old setup anymore and we really like to have Snyk in there 😉

**Comment by aarlaud** [comment]
We'll get to it asap. It might not be before next week though

**Comment by braghettos** [comment]
Waiting as well for the update!

**Comment by akibabu** [comment]
Also waiting for this

**Comment by aarlaud** [comment]
Fixed in 1.1.0. Thank you for your patience ! Also I recommend to enable [critical severity level support](https://snyk.io/blog/early-access-to-snyk-features-snyk-preview/) !
